Servers/swagger.yaml

```
@@ -1889,6 +1889,834 @@ paths:
+          application/json:
+            schema:
+              type: object
+              properties:
+                controlId:
+                  type: number
+                  example: 1
+                status:
+                  type: string
+                  anyOf:
+                    - type: string
+                    enum:
```

yamllint

[warning] wrong indentation: expected 22 but found 20

(indentation)